dataplicity/m2mmanager.py

```diff
@@ -128,16 +128,22 @@ def run(self):
                 log.exception('failed to set_identity')
 
             with self.lock:
+
+                if abs(self.m2m_client.time_since_last_packet) > 300:
```

Comment:
The clock check thread has largely replace the need for this check, but I'll leave it in as a sanity check.